src/features/emojiMod.ts

```diff
@@ -71,22 +71,19 @@ export const reactionHandlers: ReactionHandlers = {
     reportUser({ reason, message, staff, members });
   },
   "🔍": async ({ message, usersWhoReacted }) => {
-    const NUMBER_OF_AUTHORITATIVE_REACTIONS_REQUIRED_FOR_DELETION = 1;
-    const NUMBER_OF_REACTIONS_REQUIRED_FOR_DELETION = 2;
+    const KNOWN_REACTOR_THRESHOLD = 1;
+    const UNKNOWN_REACTOR_THRESHOLD = 2;
 
-    const [unauthoritativeUsersWhoReacted, authoritativeUsersWhoReacted] =
-      partition(
-        (userWhoReacted) =>
-          isStaff(userWhoReacted) || isHelpful(userWhoReacted),
-        usersWhoReacted,
-      );
+    const [knownReactors, unknownReactors] = partition(
```

Comment:
Are these the wrong way around? I think it partitions items that match the predicate into the second (or first if we're zero-based) array and I'd expect the helpful/staff partition to be named as known and the regular users as unknown but perhaps I'm looking at it wrong (apologies if so).

Reply:
You're right! Fixed